src/components/navigation/side-nav/__snapshots__/side-nav.test.tsx.snap

```diff
@@ -142,6 +165,29 @@ exports[`SideNav Component - Snapshot Test matches the snapshot in mobile view 1
               </h3>
             </div>
           </li>
+          <li
+            class="MagentaA11y__side-nav--item"
+          >
+            <div
+              class="MagentaA11y-accordion"
+            >
+              <h3
+                class="MagentaA11y-accordion__heading"
+              >
+                <button
+                  aria-expanded="false"
+                  class="MagentaA11y-accordion__headline"
+                  type="button"
+                >
+                  <span
+                    class="MagentaA11y-accordion__headline--text"
+                  >
+                    Form
+                  </span>
+                </button>
+              </h3>
+            </div>
+          </li>
```

Comment:
Include ## Further reading with the following resource:

- [WebAIM Accessible Form validation and error recovery](https://webaim.org/techniques/formvalidation/)